appinventor/components-ios/src/RetValManager.m

```diff
@@ -69,6 +69,13 @@ - (void)appendReturnValue:(NSString *)item forBlock:(NSString *)blockId
   [self addResult:output];
 }
 
+- (void)appendLogValue:(NSString *)item forBlock:(NSString *)blockId 
+  withStatus:(NSString *)status withLevel:(NSString *)level {
+    if (!item) item = @"";
```

Comment:
Note that in the header you mark this value as `nonnull`. If this isn't null, this seems superfluous?